build/config/compiler/BUILD.gn

```diff
@@ -296,6 +296,7 @@ config("compiler") {
       # Reduces global namespace pollution.
       "-s",
       "MODULARIZE=1",
+      "-sEXPORT_ES6",
```

Comment:
Why not

```
"-s",
"EXPORT_ES6",
```

Like the other options? (is it because it doesn't have `=`?)

Reply:
I think I just made an arbitrary choice. I can make it consistent with the others.